### PR TITLE
created new template cisco ios show environment temperature plus raw …

### PR DESCRIPTION
**Pull request by socretz**
…and parsed tested

##### ISSUE TYPE

 - New Template Pull Request
 - Bugfix Pull Request
 - Additional Testing
 - Docs Pull Request

##### SUMMARY

```

```

### CODE REVIEW

**Comment by socretz** [comment]
New Cisco IOS template show environment temperature

First open source contribution and fairly new coder.  Be kind :)

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py36
+envlist = py37
```
Comment (FragmentedPacket) [inline tox.ini]: Change it back to py36. You can use a few different ways of overriding to fit your Python version.

https://tox.readthedocs.io/en/latest/config.html#conf-envlist

File: templates/cisco_ios_show_environment_temperature.template [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  ^Temperature\s+State:\s+${INLET_TEMPERATURE_STATE}\s*$$
+  ^Yellow\s+Threshold\s+:\s+${INLET_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
+  ^Red\s+Threshold\s+:\s+${INLET_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Hotspot
+ 
+Hotspot
+  ^Hotspot\s+Temperature\s+Value:\s+${HOTSPOT_TEMPERATURE_VALUE}\s+Degree\s+Celsius\s*$$
+  ^Temperature\s+State:\s+${HOTSPOT_TEMPERATURE_STATE}\s*$$
+  ^Yellow\s+Threshold\s+:\s+${HOTSPOT_YELLOW_THRESHOLD}\s+Degree\s+Celsius\s*$$
+  ^Red\s+Threshold\s+:\s+${HOTSPOT_RED_THRESHOLD}\s+Degree\s+Celsius\s*$$ -> Record Start
+
```
Comment (FragmentedPacket) [inline templates/cisco_ios_show_environment_temperature.template]: Add the following to each state:
```
  ^\s*$$
  ^. -> Error
```
This helps make sure we're accounting for all output we're expecting to parse

File: templates/cisco_ios_show_environment_temperature.template [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Value INLET_TEMPERATURE_STATE (\w+)
+Value INLET_YELLOW_THRESHOLD (\d+)
+Value INLET_RED_THRESHOLD (\d+)
+Value HOTSPOT_TEMPERATURE_VALUE (\d+)
+Value HOTSPOT_TEMPERATURE_STATE (\w+)
+Value HOTSPOT_YELLOW_THRESHOLD (\d+)
+Value HOTSPOT_RED_THRESHOLD (\d+)
+
+Start
+  ^Switch\s+${SWITCH}:\s+SYSTEM\s+TEMPERATURE\s+is\s+${SWITCH_TEMPERATURE}\s*$$
```
Comment (FragmentedPacket) [inline templates/cisco_ios_show_environment_temperature.template]: I would add the following under L12
```
  ^Switch -> Continue.Record
```
And change L25 from `Record Start` to `Start`. This allows us to capture temperatures of other switches properly.

**Comment by FragmentedPacket** [comment]
Duplicate - Thorough feedback on #536

**Pull request closed without merging**